src/server/metrics.rs

```diff
@@ -332,6 +332,11 @@ lazy_static! {
             exponential_buckets(1f64, 5f64, 10).unwrap()
         )
         .unwrap();
+    pub static ref CPU_CORES_QUOTA: IntGauge = register_int_gauge!(
```

Comment:
I think a `Gauge` is better because CPU cores quota set in cgroup can be a fraction.

Reply:
At the time of this PR, the SysQuota returns an integer quota. I will follow up on your changes.